Documentation/Assets.md

```diff
@@ -2,7 +2,12 @@
 
 ## Input
 
-The assets parser accepts one (or more) asset catalogs, which it'll parse for `image set`s and folders.
+The assets parser accepts one (or more) asset catalogs, which it'll parse for supported set types and groups. We currently support the following types:
+- Color Set Type (colorset)
+- Group Type
```

Comment:
nitpicking: feels strange to have the "Group Type" in that list be in the middle of the two others, while it's kind of a special case (no extension). Maybe rather put it either at the very beginning or the very end of that list?